src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```

I don't think we need it as `static final `

Reply:
когда мы объявляем переменную private static final это константа. Я не думаю, что Random тут нужен в качестве константы. Давай сделаем его просто private